Gas Station docs updated (#446)
0xPolygon/polygon-docs: docs/tools/gas/polygon-gas-station.md

- Amoy testnet: https://gasstation-testnet.polygon.technology/amoy
- zkEVM Cardona: https://gasstation-testnet.polygon.technology/zkevm/cardona

> For fetching gas price recommendations on Amoy Testnet, below is the guide to do so

Reviewer: ```suggestion
Use the Amoy testnet below to get gas price recommendations:
```

- Amoy testnet: https://gasstation-testnet.polygon.technology/amoy
- zkEVM Cardona: https://gasstation-testnet.polygon.technology/zkevm/cardona

For fetching gas price recommendations on Amoy Testnet, below is the guide to do so

> 

Reviewer: ```suggestion
```
